application.conf

```diff
@@ -45,3 +45,9 @@ graphite {
 
 metricsPollingIntervalInSeconds = 30
 metricsPollingIntervalInSeconds = ${?INTERVAL_IN_SEC}
+
+white-list-regex = ".*"
+white-list-regex = ${?WHITE_LIST_REGEX}
+
+black-list-regex = "$a"
```

Comment:
The default value is $a?

Reply:
yes, it's a regex that matches nothing (match "a" after the end of the string) so it will filter nothing by default